README.md

```diff
@@ -84,170 +98,172 @@ This code will generate the following terminal output on success:
 You can also run:
 ```shell
 python3 recognizer_stream.py --help
-```
-  
+```  
 to list all the available options.
 
-### Synthesizer (GRPC)
+### Client flags for speech recognition (CSR)
```

Comment:
We decided that commercialy we would talk about STT - Speech-to-Text, so let me suggest to change this

Reply:
You´re right, I will update the document so that it reflect STT and Speech-To-Text everywhere in the text to have a standard way of calling it.